packages/ia-components/sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify-main.jsx

```diff
@@ -66,16 +66,10 @@ class AudioPlayerWithYoutubeSpotify extends Component {
     const albumData = flattenAlbumData(albumMetadata, playFullIAAudio);
     this.state = {
       albumData,
-      tracklistToShow: [],
       channelToPlay: 'archive',
+      channelInMemory:'archive',
       trackSelected: null, /* 0 = album */
     };
-
-    this.selectThisTrack = this.selectThisTrack.bind(this);
-    this.onChannelSelect = this.onChannelSelect.bind(this);
-    this.jwplayerPlaylistChange = this.jwplayerPlaylistChange.bind(this);
-    this.getSelectableChannels = this.getSelectableChannels.bind(this);
-    this.getAudioSourceInfoToPlay = this.getAudioSourceInfoToPlay.bind(this);
```

Comment:
where did all of these go?

Reply:
I turned each of those methods to the new ES6 arrow funcs.So as to retain their scope.